docs/apm-mutating-webhook.asciidoc

```diff
@@ -1,4 +1,17 @@
-// Pulls the APM mutating webhook docs from here:
-// https://github.com/elastic/apm-mutating-webhook/blob/main/docs/apm-mutating-webhook.asciidoc
+[[apm-mutating-admission-webhook]]
+= APM Attacher
 
-include::{apm-webhook-repo-dir}/apm-mutating-webhook.asciidoc[]
+preview::[]
+
+The APM attacher for Kubernetes simplifies the instrumentation and configuration of your application pods.
+
+The attacher includes a webhook receiver that modifies pods so they are automatically
+instrumented by an Elastic APM agent, and a Helm chart that manages its lifecycle within
+Kubernetes.
+
+// Adding links in the next PR
+// The attacher includes a {apm-attacher-ref}/x.html[webhook receiver] that modifies pods so they are automatically
+// instrumented by an Elastic APM agent, and a {apm-attacher-ref}/x.html[Helm chart] that manages its lifecycle within
+// Kubernetes.
```

Comment:
I have to add these links after https://github.com/elastic/docs/pull/2535 is merged.